lib/stellar_core_commander/docker_process.rb

```diff
@@ -143,6 +174,10 @@ def config
       <<-EOS.strip_heredoc
         POSTGRES_PASSWORD=#{database_password}
 
+        ENVIRONMENT=scc
+        CLUSTER_NAME=#{File.basename($opts[:recipe], '.rb')}
```

Comment:
fwiw, I'm not a *huge* fan of this, but having the recipe name bleed down through the whole process abstraction felt worse then reaching around it. Let me know if you have any ideas for other ways to do this.